### PR TITLE
bring gear up to best practices

### PR DESCRIPTION
**Pull request by kalebdfischer**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -2,25 +2,41 @@
...
 	"source": "https://github.com/flywheel-apps/openslide-to-png",
-	"url": "https://github.com/flywheel-apps/openslide-to-png",
+	"url": "https://github.com/openslide/openslide-python",
 	"custom": {
 		"gear-builder": {
 			"category": "converter",
-			"image": "openslide"
+			"image": "flywheel/openslide-to-png"
+		},
+    "docker-image": "flywheel/openslide-to-png",
```
Comment (lmperry) [inline manifest.json]: My preference is to use a tag with the docker-image `flywheel/openslide-to-png:1.0.0`. That allows me to have multiple "versions" within my local registry if desired.

File: manifest.json [hunk 2]
```diff
@@ -2,25 +2,41 @@
...
+		"flywheel":{
+			"suite": "image_tools"
```
Comment (lmperry) [inline manifest.json]: This is a hard one... Let's use "Flywheel" as the suite - or better yet poll the team.

File: manifest.json [hunk 3]
```diff
@@ -2,25 +2,41 @@
...
+				"ERROR",
+				"WARNING",
+				"INFO",
+				"DEBUG"
+			]
+		}
+	},
+	"command": "python run.py",
 	"author": "Travis Richardson",
 	"maintainer": "Travis Richardson",
```
[the remark this answers is not in the document]
Comment (kalebdfischer) [reply manifest.json]: fixed the author to the OpenSlide authors and maintainer to Flywheel

File: manifest.json (hunk 1 above)
Comment (kalebdfischer) [reply manifest.json]: Fixed.

File: manifest.json (hunk 2 above)
Comment (kalebdfischer) [reply manifest.json]: Changed suite to flywheel, but would also be good to have a general suite discussion

**Pull request merged**